src/reactapp/src/api/sendRequest.js

```diff
@@ -37,6 +46,20 @@ export default function sendRequest(
       }
       return response.json();
     })
+    .then(response => {
+      if (!responseContainErrors(response) || !responseDataEmpty(response)) {
+        return response;
+      }
+
+      const errors = _get(response, 'errors', []);
+      const exception = new GraphQLResponseException(errors);
+
+      dispatch({
+        type: SET_PAGE_MESSAGE,
+        payload: { type: 'error', message: exception.message },
+      });
+      throw exception;
+    })
     .catch(exception => {
       console.error(exception);
       throw exception;
```

Comment:
Should we also display,  "something went wrong, please try again" if an exception happens?
This would be in the case the server returns a 4xx or 5xx (so we have no response body at all).

It would be good to display an "oops" message then

Reply:
can you please take care of this?

Reply:
Okay to postpone for after this pr.